src/sass/app_2/modules/contract.scss

```diff
@@ -119,7 +119,17 @@ $CONTRACT_INFO_BOX_PADDING: 1.5em;
     transform: none;
     opacity: 1;
     transition: transform 0.4s ease, opacity 0.4s linear;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
 
+    &-button:hover {
+        cursor: pointer;
+    }
+    &-icon {
+        width: 3.2em;
```

Comment:
I still have my reservations with using `em` for `width` and `height` for icons. In some cases, it seems like the scaling is off when using this method.

Reply:
sure, i've change it to px